src/extension.ts

```diff
@@ -84,7 +84,7 @@ export function getDecorations(
     documentText: string,
     sentryProjects?: SentryProject[]
 ): sourcegraph.TextDocumentDecoration[] {
-    const params: Params = getParamsFromUriPath(documentUri)
+    const params: Params | null = getParamsFromUriPath(documentUri)
```

Comment:
```suggestion
    const params = getParamsFromUriPath(documentUri)
```

Typescript will infer this